notebooks/automatic_embedding_tei_inference_endpoints.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<p>Typos: "I didnt" -&gt; "I didn't"</p><p> </p><p>Can you explain what <code>MAX_BATCH_TOKENS</code>&nbsp;or&nbsp;<code>MAX_CONCURRENT_REQUESTS</code>mean and what they do?</p><p>Is HW - hardware? Maybe better to spell it out. Would be cool to provide some guidance or examples here on how to choose values for these. </p><p> </p><p>Oh, so this is a revision of the model repository?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>